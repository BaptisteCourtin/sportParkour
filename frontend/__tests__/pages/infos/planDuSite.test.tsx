import { render } from "@testing-library/react";
import PlanDuSite from "@/pages/infos/planDuSite";

describe("planDuSite", () => {
  it("renders correctly when loading", () => {
    const { asFragment } = render(<PlanDuSite />);
    expect(asFragment()).toMatchSnapshot();
  });
});
