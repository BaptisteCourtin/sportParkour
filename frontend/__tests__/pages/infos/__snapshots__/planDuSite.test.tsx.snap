// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`planDuSite renders correctly when loading 1`] = `
<DocumentFragment>
  <main
    class="pagesInfos"
  >
    <h1>
      PLAN DU SITE
    </h1>
    <h2>
      En vigueur au 01/08/2024
    </h2>
    <ul
      class="ulPlanSite"
    >
      <h3>
        Pour vous connecter
      </h3>
      <li>
        <a
          href="/auth/inscription"
        >
          <svg
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 256 512"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M246.6 278.6c12.5-12.5 12.5-32.8 0-45.3l-128-128c-9.2-9.2-22.9-11.9-34.9-6.9s-19.8 16.6-19.8 29.6l0 256c0 12.9 7.8 24.6 19.8 29.6s25.7 2.2 34.9-6.9l128-128z"
            />
          </svg>
          Inscription
        </a>
      </li>
      <li>
        <a
          href="/auth/login"
        >
          <svg
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 256 512"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M246.6 278.6c12.5-12.5 12.5-32.8 0-45.3l-128-128c-9.2-9.2-22.9-11.9-34.9-6.9s-19.8 16.6-19.8 29.6l0 256c0 12.9 7.8 24.6 19.8 29.6s25.7 2.2 34.9-6.9l128-128z"
            />
          </svg>
          Authentification
        </a>
      </li>
      <li>
        <a
          href="/auth/resetPassword/resetPassword"
        >
          <svg
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 256 512"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M246.6 278.6c12.5-12.5 12.5-32.8 0-45.3l-128-128c-9.2-9.2-22.9-11.9-34.9-6.9s-19.8 16.6-19.8 29.6l0 256c0 12.9 7.8 24.6 19.8 29.6s25.7 2.2 34.9-6.9l128-128z"
            />
          </svg>
          Modifier votre mot de passe
        </a>
      </li>
      <br />
      <br />
      <h3>
        Pour rechercher :
      </h3>
      <li>
        <a
          href="/"
        >
          <svg
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 256 512"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M246.6 278.6c12.5-12.5 12.5-32.8 0-45.3l-128-128c-9.2-9.2-22.9-11.9-34.9-6.9s-19.8 16.6-19.8 29.6l0 256c0 12.9 7.8 24.6 19.8 29.6s25.7 2.2 34.9-6.9l128-128z"
            />
          </svg>
          Rechercher un parkour
        </a>
      </li>
      <li>
        <a
          href="/parkour/parkourMap"
        >
          <svg
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 256 512"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M246.6 278.6c12.5-12.5 12.5-32.8 0-45.3l-128-128c-9.2-9.2-22.9-11.9-34.9-6.9s-19.8 16.6-19.8 29.6l0 256c0 12.9 7.8 24.6 19.8 29.6s25.7 2.2 34.9-6.9l128-128z"
            />
          </svg>
          Rechercher un parkour avec une carte
        </a>
      </li>
      <li>
        <a
          href="/allEpreuves"
        >
          <svg
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 256 512"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M246.6 278.6c12.5-12.5 12.5-32.8 0-45.3l-128-128c-9.2-9.2-22.9-11.9-34.9-6.9s-19.8 16.6-19.8 29.6l0 256c0 12.9 7.8 24.6 19.8 29.6s25.7 2.2 34.9-6.9l128-128z"
            />
          </svg>
          Rechercher une épreuve
        </a>
      </li>
      <br />
      <br />
      <h3>
        Votre profil (il faut avoir un compte et être connecté)
      </h3>
      <li>
        <a
          href="/user/profil"
        >
          <svg
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 256 512"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M246.6 278.6c12.5-12.5 12.5-32.8 0-45.3l-128-128c-9.2-9.2-22.9-11.9-34.9-6.9s-19.8 16.6-19.8 29.6l0 256c0 12.9 7.8 24.6 19.8 29.6s25.7 2.2 34.9-6.9l128-128z"
            />
          </svg>
          Votre profil
        </a>
      </li>
      <li>
        <a
          href="/user/favoris"
        >
          <svg
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 256 512"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M246.6 278.6c12.5-12.5 12.5-32.8 0-45.3l-128-128c-9.2-9.2-22.9-11.9-34.9-6.9s-19.8 16.6-19.8 29.6l0 256c0 12.9 7.8 24.6 19.8 29.6s25.7 2.2 34.9-6.9l128-128z"
            />
          </svg>
          Vos parkours favoris
        </a>
      </li>
      <li>
        <a
          href="/user/notes"
        >
          <svg
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 256 512"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M246.6 278.6c12.5-12.5 12.5-32.8 0-45.3l-128-128c-9.2-9.2-22.9-11.9-34.9-6.9s-19.8 16.6-19.8 29.6l0 256c0 12.9 7.8 24.6 19.8 29.6s25.7 2.2 34.9-6.9l128-128z"
            />
          </svg>
          Les parkours où vous avez mis un avis
        </a>
      </li>
      <br />
      <br />
      <h3>
        Les infos
      </h3>
      <li>
        <a
          href="/infos/cgu"
        >
          <svg
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 256 512"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M246.6 278.6c12.5-12.5 12.5-32.8 0-45.3l-128-128c-9.2-9.2-22.9-11.9-34.9-6.9s-19.8 16.6-19.8 29.6l0 256c0 12.9 7.8 24.6 19.8 29.6s25.7 2.2 34.9-6.9l128-128z"
            />
          </svg>
          CGU
        </a>
      </li>
      <li>
        <a
          href="/infos/faq"
        >
          <svg
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 256 512"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M246.6 278.6c12.5-12.5 12.5-32.8 0-45.3l-128-128c-9.2-9.2-22.9-11.9-34.9-6.9s-19.8 16.6-19.8 29.6l0 256c0 12.9 7.8 24.6 19.8 29.6s25.7 2.2 34.9-6.9l128-128z"
            />
          </svg>
          FAQ
        </a>
      </li>
      <li>
        <a
          href="/infos/mentionsLegales"
        >
          <svg
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 256 512"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M246.6 278.6c12.5-12.5 12.5-32.8 0-45.3l-128-128c-9.2-9.2-22.9-11.9-34.9-6.9s-19.8 16.6-19.8 29.6l0 256c0 12.9 7.8 24.6 19.8 29.6s25.7 2.2 34.9-6.9l128-128z"
            />
          </svg>
          Mentions légales
        </a>
      </li>
      <li>
        <a
          href="/infos/planDuSite"
        >
          <svg
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 256 512"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M246.6 278.6c12.5-12.5 12.5-32.8 0-45.3l-128-128c-9.2-9.2-22.9-11.9-34.9-6.9s-19.8 16.6-19.8 29.6l0 256c0 12.9 7.8 24.6 19.8 29.6s25.7 2.2 34.9-6.9l128-128z"
            />
          </svg>
          Plan du site (c'est cette page)
        </a>
      </li>
      <li>
        <a
          href="/infos/politiqueDeConfidentialite"
        >
          <svg
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 256 512"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M246.6 278.6c12.5-12.5 12.5-32.8 0-45.3l-128-128c-9.2-9.2-22.9-11.9-34.9-6.9s-19.8 16.6-19.8 29.6l0 256c0 12.9 7.8 24.6 19.8 29.6s25.7 2.2 34.9-6.9l128-128z"
            />
          </svg>
          Politique de confidentialité
        </a>
      </li>
      <li>
        <a
          href="/infos/quiSommesNous"
        >
          <svg
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 256 512"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M246.6 278.6c12.5-12.5 12.5-32.8 0-45.3l-128-128c-9.2-9.2-22.9-11.9-34.9-6.9s-19.8 16.6-19.8 29.6l0 256c0 12.9 7.8 24.6 19.8 29.6s25.7 2.2 34.9-6.9l128-128z"
            />
          </svg>
          Qui sommes nous
        </a>
      </li>
      <br />
      <br />
      <h3>
        La meilleure page du site
      </h3>
      <li>
        <a
          href="/404"
        >
          <svg
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 256 512"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M246.6 278.6c12.5-12.5 12.5-32.8 0-45.3l-128-128c-9.2-9.2-22.9-11.9-34.9-6.9s-19.8 16.6-19.8 29.6l0 256c0 12.9 7.8 24.6 19.8 29.6s25.7 2.2 34.9-6.9l128-128z"
            />
          </svg>
          La page 404
        </a>
      </li>
    </ul>
  </main>
</DocumentFragment>
`;
